src/jl_uv.c

```diff
@@ -89,6 +107,7 @@ static void jl_uv_closeHandle(uv_handle_t *handle)
     // also let the client app do its own cleanup
     if (handle->type != UV_FILE && handle->data) {
         size_t last_age = jl_get_ptls_states()->world_age;
+        // TODO: potential data race
```

Comment:
Do you mean `jl_world_counter`? That is a race but you shouldn't add the todo in this PR (you can of course add to your local branch but not in a unrelated PR.)

Reply:
I'll do it in a separate revision since there are a lot of changes related to this race